src/main/scala/com/couchbase/spark/Logging.scala

```diff
@@ -0,0 +1,135 @@
+package com.couchbase.spark
+
+/**
+  * Created by shivansh on 8/8/16.
+  */
+
+import org.apache.log4j.{Level, LogManager, PropertyConfigurator}
+import org.slf4j.{Logger, LoggerFactory}
+import org.slf4j.impl.StaticLoggerBinder
+
+trait Logging {
```

Comment:
This logging trait is not gone but its made private for internal use of spark only from spark 2.0 . You can take a look here https://issues.apache.org/jira/browse/SPARK-9307